src/index.js

```diff
@@ -129,21 +129,35 @@ export async function showLocation(options) {
     case 'google-maps':
       const googleDirectionMode = getDirectionsModeGoogleMaps();
       // Always using universal URL instead of URI scheme since the latter doesn't support all parameters (#155)
-      url = 'https://www.google.com/maps/dir/?api=1';
       if (useSourceDestiny) {
```

Comment:
Should we maybe for backwards compatibility also go into this branch of the logic if a `directionsMode` is defined?

```suggestion
      if (useSourceDestiny || options.directionsMode) {
```